parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -52,12 +52,6 @@ fi
 configureHTTPProxyCA
 {{- end}}
 
-{{- if EnableChronyFor1804}}
```

Comment:
A follow up after this is to remvoe this EnableChronyFor1804 flag from AgentBaker and RP.